### ISSUE TITLE
5.0.2 introduces a bug

### ISSUE THREAD

**Issue by iamvdo** [postcss/postcss-import#25]
I don't really know the problem, but let's take 2 files:

```css
/*file 1*/
@import 'file2.css';
.file1 {}
```
```css
/*file 2*/
.file2--1 {}
.file2--2 {}
```

And add a `console.log(styles.nodes[1].parent)` on line `index.js:67`.

When using `postcss-import@5.0.1`, you get 3 nodes in `parent`, while with `postcss-import@5.0.2`, you only get 2 nodes, only those from `file2.css`. I checked your modifications, but my brain doesn't find where the problem is.

And this breaks some tests when using postcss-import with other modules.

**Comment by MoOx** [comment]
Then fun part is that `styles.nodes[*].parent` should be `===` to `styles`.
I will find what I break. Thanks for reporting.

**Comment by MoOx** [comment]
Oh I just forgot to update imported nodes parents... Fixed and released as 5.0.3.

**Comment by iamvdo** [comment]
:+1:
